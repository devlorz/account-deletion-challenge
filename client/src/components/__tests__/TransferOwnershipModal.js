import React from 'react'
import { render } from 'react-testing-library'

import TransferOwnershipModal from '../TransferOwnershipModal'

test('snapshot', () => {
  const { container } = render(
    <TransferOwnershipModal
      nextPage={jest.fn()}
      loading={true}
      disabledNextPage={true}
    >
      Test
    </TransferOwnershipModal>
  )
  expect(container.firstChild).toMatchSnapshot()
})
