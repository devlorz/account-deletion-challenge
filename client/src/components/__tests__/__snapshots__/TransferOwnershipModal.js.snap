// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<div
  data-reactroot=""
>
  <h1>
    Transfer ownership
  </h1>
  <p>
    Before you leaving, it is required to transfer your tasks, projects and workspace admin rights to other person.
  </p>
  <div>
    Loading...
  </div>
  <button
    disabled=""
  >
    Next
  </button>
</div>
`;
